src/funcade/codec.clj

```diff
@@ -37,10 +37,10 @@
         {:funcade/error ex}))))
 
 (defn decode64 [^String s]
-  (.decode (Base64/getDecoder) s))
+  (.decode (Base64/getUrlDecoder) s))
 
 (defn encode64 [^bytes bs]
-  (.encodeToString (Base64/getEncoder) bs))
+  (.encodeToString (Base64/getUrlEncoder) bs))
```

Comment:
this function is unused but I figured I would update it for consistency. I did not delete it because `inflate!`, `gzip!`, `gunzip!` and `encode64` were all unused and, if we desire cleanup, we can do so as a separate commit